app/controllers/v1/analysis_responses_controller.rb

```diff
@@ -0,0 +1,37 @@
+class V1::AnalysisResponsesController < ApplicationController
+  before_action :authenticate_user!
+
+  def create
+    @response = Response.find_or_create_by(responder: analysis)
```

Comment:
If you're calling `.save` below, using `Response.find_or_initialize_by` could save you a query.